kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -54,26 +80,166 @@
           this.isFullscreen = true;
         }
       },
+      getPage(pageNum) {
+        return this.pdfDocument.getPage(pageNum);
+      },
+      displayPage(pdfPage) {
+        return new Promise((resolve, reject) => {
```

Comment:
Display a page, but add in lots of potential escape hatches if rendering has been cancelled in the mean time - isn't async fun?